_includes/cohort-data.html

```diff
@@ -0,0 +1,34 @@
+{% assign cohort = include.cohort %}
+{% assign people = site.data.people %}
+{% assign projects = site.data.cohorts.[cohort].projects %}
+{% assign metadata = site.data.cohorts.[cohort].metadata %}
```

Comment:
This variable is not used later so you can remove it

Reply:
True. 💯 Thank you!

Reply:
**Update:** It turns out this variable is actually being used to display the experts and organisers on the _https://openlifesci.org/ols-x.html_ pages. 😄 I'm putting it back now.